### ISSUE TITLE
Can't sync with DroneCore-Proto submodule on Ubuntu

### ISSUE THREAD

**Issue by hamishwillee** [dronecore/DroneCore-Python#1]
I get the following error: 
```
~/github/dronecore/DroneCore-Python$ git submodule update
Cloning into 'proto'...
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
fatal: clone of 'git@github.com:dronecore/DroneCore-Proto.git' into submodule path 'proto' failed
```

I can separately clone  DroneCore-Proto repo

**Comment by hamishwillee** [comment]
Yes, that works. Sadly the very next step in README  fails #5

**Comment by JonasVautherin** [comment]
Should be fixed by #4, but it is not merged yet.
